stockticker/stockticker.py

```diff
@@ -1,20 +1,48 @@
 """stockticker app"""
+from datetime import datetime, timedelta
 import requests
+from requests import Timeout
 import simplejson as json
 from flask import Flask, render_template
 from flask import  request # pylint: disable=W0611
 import pandas as pd
 from bokeh.plotting import figure
 from bokeh.embed import components
 
-def create_app(prophet_url, secret_key, bokeh_version):
+class QuandlException(Exception):
+    """Exception for Quandl API"""
+    pass
+
+def query_quandl(ticker, quandl_key, value='open', days=100):
+    """get stock value from quandl"""
+    days = min(days, 800) # getting much larger than this causes too long URI
+
+    # prepare date string for the Quandl API
+    today = datetime.today()
+    dates = ','.join((today-timedelta(days=i)).strftime('%Y-%m-%d') for i in range(1, days+1))
```

Comment:
is there a way to specify start_date and end_date instead?

Reply:
ah, I just learned about the different API call for this... I'll make the changes! It'll make cleaner.